yaksh/live_server_tests/selenium_test.py

```diff
@@ -115,6 +116,17 @@ def open_quiz(self):
         self.test_python_question(question_label=5)
         self.test_bash_question(question_label=4)
 
+    def quit_quiz(self):
+        quit_link_elem = WebDriverWait(self.driver, 5).until(
```

Comment:
This wait business can be encapsulated into a function so you don't need to keep repeating the same `WebDriverWait(self.driver, 5).until(...)`